### ISSUE TITLE
C3 Crash !

### ISSUE THREAD

**Issue by DieppedalleDavid** [Scirra/Construct-3-bugs#2768]
## Problem description

Good evening,

Contruct3 crash when I close the previews in test mode.
I don't know why.

## Attach a .c3p

https://drive.google.com/open?id=1ZrxMqTxMBhgLSkXg9HROJIhxY4a8mZO7

## Steps to reproduce
Start the game in debug mode, close the game with the cross, sometimes the game crash and Construc3 with !
Try it several times, it's not every time!

**Affected browsers/platforms:** 
Construct 3 desktop - 142.2, Windows 7, Chrome

**First affected release:** 

## System details

In the Construct 3 folder, there is a file named Debug.txt, there is this for example:
0316/003852.926:WARNING:minidump_to_upload_parameters.cc(66)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
0316/003852.952:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)
0319/192749.161:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192751.060:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192751.309:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192751.486:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192752.005:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192752.416:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192752.728:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192753.088:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192753.455:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192754.457:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192754.666:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192755.435:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192755.695:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0319/192755.948:ERROR:registration_protocol_win.cc(56)] CreateFile: The specified file is not found. (0x2)
0324/193001.599:WARNING:minidump_to_upload_parameters.cc(66)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
0324/193001.682:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)
0325/012708.693:WARNING:minidump_to_upload_parameters.cc(66)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
0325/012708.737:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)
0325/012912.879:WARNING:minidump_to_upload_parameters.cc(66)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
0325/012912.906:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)
0325/013211.642:WARNING:minidump_to_upload_parameters.cc(66)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
0325/013211.670:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)
0325/013212.432:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)
0325/013526.494:WARNING:minidump_to_upload_parameters.cc(66)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
0325/013526.523:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)
0325/014000.769:WARNING:minidump_to_upload_parameters.cc(66)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
0325/014000.797:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)
0325/014903.870:WARNING:minidump_to_upload_parameters.cc(66)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
0325/014903.911:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)
0325/015420.223:WARNING:minidump_to_upload_parameters.cc(66)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
0325/015420.251:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The operation was successful. (0x0)

**Comment by DieppedalleDavid** [comment]
I would add that, at the moment, I have big problems with construction 3 (office).
It takes a long time to load it (game), when I want to test it.

So all these problems are not present on the online version!
Construct 3 on the Internet works very well, no crashes!

**Comment by AshleyScirra** [comment]
I can't reproduce here: I opened the project in the latest desktop download, debugged and closed with the X in the top-right 5 times, and it worked fine every time. It's not clear which version you're using; please make sure you're using the latest desktop download.

**Comment by DieppedalleDavid** [comment]
Well, try again !
Me with the latest version is the same !

**Comment by DieppedalleDavid** [comment]
And what does that mean?:

[0325/181614.387:WARNING:minidump_to_upload_parameters.cc(67)] duplicate annotation name metrics_client_id, discarding value 0c3b3a17f3934785a859920f34eb2feb
[0325/181614.417:ERROR:http_transport_win.cc(178)] WinHttpCrackUrl: L’opération a réussi. (0x0)

**Comment by DieppedalleDavid** [comment]
Here is a video, I unfortunately, as if by chance, could not reproduce the bug!
Well the video is not interesting, there is nothing special, look at the first few minutes, no need to watch all of them....

I don't understand why it's so slow to launch, sometimes it doesn't launch, I think it comes from a problem with my internet connection, I have problems with the internet right now, it doesn't work normally...

I find it very unfortunate that C3 is not an "Offline" mode.
If you don't have internet, it doesn't start!
I'm talking about the "Office" version!

We'd say C3 is waiting for something before this throw, I think he's waiting for clearance, right?

In short, C3 makes me more desperate every day!

https://drive.google.com/open?id=1Gou1gWaMIYqf5nOxU8TQp0DUUCrI57LI

**Comment by AshleyScirra** [comment]
Sorry, there is nothing we can do if we can't reproduce the issue ourselves, which we need reliable steps to reproduce to do. Please see the bug report guidelines - videos generally don't help either. Also the logs you have provided so far aren't any help.
